dev_guide/managing_images.adoc

```diff
@@ -477,8 +477,9 @@ $ oc secrets new-dockercfg <pull_secret_name> \
     --docker-password=<password> --docker-email=<email>
 ----
 
-To use a secret for pulling images for pods, you must add the secret to your
-service account. The name of the service account in this example should match
+If `serviceAccountConfig.limitSecretReferences` is set to `true` in the master
```

Comment:
@jim-minter OK. I'm still a bit confused why this was mentioned in the initial BZ, but I've reverted it. Let me know if this still is not what you're requesting.